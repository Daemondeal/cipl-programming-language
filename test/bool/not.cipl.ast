(expression (call print (not true)))
(expression (call print (not false)))
(expression (call print (not (not true))))

