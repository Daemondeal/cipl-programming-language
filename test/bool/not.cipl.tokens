Token          Lexeme         Line 
IDENTIFIER     print          1    
LEFT_PAREN     (              1    
NOT            not            1    
TRUE           true           1    
RIGHT_PAREN    )              1    
LINE_END       \n             2     0
IDENTIFIER     print          2    
LEFT_PAREN     (              2    
NOT            not            2    
FALSE          false          2    
RIGHT_PAREN    )              2    
LINE_END       \n             3     0
IDENTIFIER     print          3    
LEFT_PAREN     (              3    
NOT            not            3    
NOT            not            3    
TRUE           true           3    
RIGHT_PAREN    )              3    
LINE_END       \n             3    
EOF                           3    
