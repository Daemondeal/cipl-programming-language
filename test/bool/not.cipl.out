::stdout::
false
true
true

::stderr::
